include/ringmesh/basic/task_handler.h

```diff
@@ -40,6 +40,8 @@
 #include <ringmesh/basic/common.h>
 #include <ringmesh/basic/types.h>
 
+#include <functional>
```

Comment:
We try to keep our includes in an ordered way. First are the system includes. In this case, could you move your include before <future> to also follow alphabetical order?